docs/csharp/tuples.md

```diff
@@ -26,24 +26,26 @@ different ways to use them, and initial guidance on working with Tuples.
 
 Let's start with the reasons for adding new Tuple support. Methods return
 a single object. Tuples enable you to package multiple values in that single
-object more easily.
+object more easily. You don't need behavior on these types, but you want to
+package multiple values in a single object.
 
-The .NET Framework already had a `Tuple` generic classes. These classes,
-however, suffered from two limitations. For one, the `Tuple` classes named
+The .NET Framework already had `Tuple` generic classes. These classes,
```

Comment:
Tuple generic classes, or generic Tuple classes?
Also, I don't like the use of "had" (past perfect); better "has" (present perfect), since they still exist.